tests/unit/fork_test.cpp

```diff
@@ -0,0 +1,52 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+#include <unistd.h>
```

Comment:
Better, but just out of curiosity: why do we need unistd.h in this specific file?